packages/client/hmi-client/src/components/asset/tera-asset.vue

```diff
@@ -166,6 +162,23 @@ function onScroll(event: Event) {
 	chosenItem.value = closestItem;
 }
 
+function onRename(newName: string) {
+	if (!newName) return;
```

Comment:
I found myself renaming asset to an empty string because I pressed the return key to fast... 
```suggestion
	if (!newName || isEmtpy(newName)) return;
```